mlflow/gateway/provider_registry.py

```diff
@@ -62,6 +62,10 @@ def _register_plugin_providers(registry: ProviderRegistry):
         registry.register(p.name, cls)
 
 
+def is_supported_provider(name: str):
+    return name in provider_registry.keys()
```

Comment:
```suggestion
def is_supported_provider(name: str) -> bool:
    return name in provider_registry
```

Reply:
`keys()` is required as `provider_registry` is not a dictionary